LayoutTests/platform/mac/TestExpectations

```diff
@@ -1798,7 +1798,7 @@ webkit.org/b/209692 [ BigSur+ ] platform/mac/fast/text/text-security-disc-bullet
 webkit.org/b/209692 [ BigSur+ ] fast/text/text-security-disc-bullet-pua.html [ ImageOnlyFailure ]
 
 # rdar://63094384 (REGRESSION: (r261413): [ Guard-Malloc ] webgl/2.0.0/conformance2/textures/misc/tex-unpack-params.html is crashing consistently with - libGLImage.dylib: void glgConvertTo_32<GLGConverter_RGB8_ARGB8, (GLGMemory)1>)
-[ BigSur+ ] webgl/2.0.0/conformance2/textures/misc/tex-unpack-params.html [ Skip ]
+[ BigSur+ Guard-Malloc ] webgl/2.0.0/conformance2/textures/misc/tex-unpack-params.html [ Skip ]
```

Comment:
Looking at the bug, it's weird, and I'm not sure if we even need to skip this under GuardMalloc any more. This does look like a stop in the right direction of sure.

Nit:  BigSur+ is not needed any more.